gp-address-autocomplete/gpaa-show-place-name.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Gravity Perks // Address Autocomplete // Search & Show by Place Name
+ * https://gravitywiz.com/documentation/gravity-forms-address-autocomplete/
+ *
+ * Find addresses by their name. For example, "Mount Trashmore" in Virginia Beach would resolve to
+ * an address of "310 Edwin Drive, Virginia Beach, VA 23462".
+ *
+ * Developer's Note: The JS in this snippet cannot be included via the Custom Javascript plugin due
+ * to an order-of-events issue with GPAA calling the options filter before the Custom Javascript plugin
+ * has initialized.
+ */
+// Update "123" to your form ID or remove "_123" to apply to all forms.
+add_action( 'gform_pre_enqueue_scripts_123', function() {
+	?>
+	<script>
+		// Enable search by Place Name
+		gform.addFilter( 'gpaa_autocomplete_options', function( options ) {
+			options.types = [ 'geocode', 'establishment' ];
+			return options;
```

Comment:
```suggestion
			options.types = [ 'geocode', 'establishment' ];
			options.fields.push( 'name' );
			return options;
```

This will allow you to skip https://github.com/gravitywiz/gp-address-autocomplete/pull/36 and immediately ship this to the customer without needing to update GPAA.